config/application.yml.default

```diff
@@ -368,6 +374,8 @@ development:
   hmac_fingerprinter_key_queue: '["11111111111111111111111111111111", "22222222222222222222222222222222"]'
   identity_pki_local_dev: true
   in_person_proofing_enabled: true
+  inherited_proofing_enabled: true
+  va_inherited_proofing_mock_enabled: true
```

Comment:
these have been removed from main so we should probably remove

Reply:
Removed. This likely occurred due to a minor difference in the source branch when I used `git checkout --merge ...`